### PR TITLE
Add AUTHORS file from the public git log infos. Fixes #888

### LINKED ISSUE #888: AUTHORS file missing
The Developers guide [mentions](https://github.com/certtools/intelmq/blob/f29ef52c488e141dceb8e613668c2acae77b3927/docs/Developers-Guide.md#license-and-author-files) a AUTHORS file, but no such file exists.

### PR DESCRIPTION
**Pull request by aaronkaplan**
ok?

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Tomás Lima<synchroack@protonmail.ch>
```
Comment (wagner-certat) [inline AUTHORS]: Space between name and < missing.

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+alexey 
+cert-lv <dev@cert.lv>
+cokebottle <cok3bottle@gmail.com>
+jgedeon120 <joe.gedeon@gmail.com>
+kralca 
+ondj <j.onderka@nbu.cz>
+pkug <patrikas.kugrinas@ktu.lt>
+slashtolau <tolau@protonmail.com>
+ufoczek <ufoczek@gmail.com>
+
```
Comment (wagner-certat) [inline AUTHORS]: two newlines at end of file

**Comment by aaronkaplan** [comment]
travis build fail will be fixed in a different PR. Can you merge pls?

**Comment by wagner-certat** [comment]
> travis build fail will be fixed in a different PR.

Merged master into this branch.

> Can you merge pls?

Still waiting for @SYNchroACK's review

**Pull request merged**